pulsar-broker/src/test/java/org/apache/pulsar/broker/service/AdvertisedAddressTest.java

```diff
@@ -42,7 +42,7 @@
 
     @BeforeMethod
     public void setup() throws Exception {
-        bkEnsemble = new LocalBookkeeperEnsemble(3, 0, () -> 0);
+        bkEnsemble = new LocalBookkeeperEnsemble(1, 0, () -> 0);
```

Comment:
a change to this file is not necessary in the scope of this PR, btw we can save to start 2 bookies for this little test